Dockerfile

```
@@ -7,4 +7,8 @@ COPY operator /bin/operator
 # On busybox 'nobody' has uid `65534'
 USER 65534
 
+LABEL org.opencontainers.image.source="https://github.com/prometheus-operator/prometheus-operator" \
+    org.opencontainers.image.documentation="https://prometheus-operator.dev/docs/prologue/introduction/" \
```

(nit) to avoid HTTP 404 replies if the page moves in the future.  
```suggestion
    org.opencontainers.image.documentation="https://prometheus-operator.dev/" \
```

---

Updated :+1: